qa/rpc-tests/walletbackup.py

```diff
@@ -136,7 +136,7 @@ def run_test(self):
 
         # Generate 101 more blocks, so any fees paid mature
```

Comment:
I think there also needs to be a sync at the end of do_one_round:

#Must sync mempools before mining
self.nodes[3].generate(1)
+self.sync_blocks()

Reply:
merged your PR